README.md

```diff
@@ -38,20 +24,27 @@ yarn cli <options>
 
 e.g. `yarn cli -I uart -d /dev/ttyUSB0`
 
-Step 3: before and after development, the existing tests should be run:
+Step 2: before and after development, the existing tests should be run:
 
 ```
 yarn test_uhf <options>
 yarn test_sband <options>
 ```
 
-### Troubleshooting
+## Using This Repository with a UHF SDR
 
-On some systems, you may see something like:
+If using this repository for FlatSat testing or deployment, follow these slightly different instructions. Hardware setup steps are excluded.
 
-`00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'build'`
+Step 1: Follow [Ground Station Software Setup Guide](https://docs.google.com/document/d/1-BGoCOlOCisQk08cB9kHaWAGk-gdsx6xPlfZvo5EFBs/edit). Run the GNURadio flow graph you will be using for testing or operations.
```

Comment:
Documentation should be self contained

Reply:
It's documentation that's pertinent to more than just this repository. Why make it contained?

Reply:
It's not private information

Reply:
Whoops. Try now, it should be public.

Reply:
I don't see any reason why that documentation can't be here, making the repository and its docs self contained

Reply:
Also, that looks like it could be a script

Reply:
It can still act as a reference for other projects being here. Self contained documentation is useful for using the repository it's in, it doesn't have to be unique. It can also be copied to ex2_sdr if you want

Reply:
I'd prefer not to have to change multiple locations if setup steps change. I still don't understand your reasoning for disallowing my preferred way of documenting this. Your last message seemed to disregard the reasoning I outlined in my previous message.
> Self contained documentation is useful for using the repository it's in
Could you clarify this? This doesn't help me understand why your proposed way of doing this is advantageous.

Reply:
New developers seeing this repository for the first time will have no idea how any of this works. Going to another domain to get the other half of the instructions for using the software is confusing. Also, there is no guarantee that document still exists in the future and it may have too much information in later. Having the documentation self contained means the documentation is always current for any version of the software, and does not involve searching for the full story. 

I did not disregard your previous message, I said you can copy the build steps to there. And these steps can still be used as reference for anyone doing development being in this readme.